package.json

```diff
@@ -43,7 +43,7 @@
     "prettycron": "0.10.0",
     "query-string": "4.1.0",
     "react": "0.14.7",
-    "react-ace": "3.4.1",
+    "react-ace": "3.7.0",
```

Comment:
Have you checked the implications of this update with the old flow?
Does that still work?